### PR TITLE
Example: time test example for random.normal distr.

### PR DESCRIPTION
**Pull request by samir-nasibli**

### CODE REVIEW

File: dpnp/backend/examples/example10.cpp [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+
+    size_t seed = 10;
+    double loc = 0.0;
+    double scale = 1.0;
+
+    std::cout << "Normal distr. params:\nloc is " << loc << ", scale is " << scale << std::endl;
+
+    for (size_t i = 0; i < iters; ++i)
+    {
+        dpnp_rng_srand_c(seed);
```
Comment (shssf) [inline dpnp/backend/examples/example10.cpp:60]: Are you sure you need to set the same initial value at any loop iteration?

**Review by shssf — APPROVED** [review]

File: dpnp/backend/examples/example10.cpp [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+// CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE)
+// ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF
+// THE POSSIBILITY OF SUCH DAMAGE.
+//*****************************************************************************
+
+/**
+ * Example 10.
+ *
+ * Possible compile line:
+ * clang++ -fPIC -fsycl dpnp/backend/examples/example10.cpp -Idpnp -Idpnp/backend/include -Ldpnp -Wl,-rpath='$ORIGIN'/dpnp -ldpnp_backend_c -o example10 -lmkl_sycl -lmkl_intel_ilp64 -lmkl_core
```
Comment (shssf) [inline dpnp/backend/examples/example10.cpp:30]: I suspect you don't have to set `-fsycl` here. Also, it might be a regular `gcc` as a compiler

Comment (shssf) [reply dpnp/backend/examples/example10.cpp:30]: Please see #754 for details

**Comment by shssf** [comment]
Please remove the "in progress" label when this PR is ready to be merged.

**Comment by samir-nasibli** [comment]
> Please remove the "in progress" label when this PR is ready to be merged.

@shssf This PR is ready to merge.

**Pull request merged**